front/app/containers/Admin/projects/project/analysis/Insights/Question.tsx

```diff
@@ -119,10 +126,10 @@ const Question = ({ insight }: Props) => {
         >
           {hasFilters && (
             <>
-              <Box>Summary for:</Box>
+              <Box>Summary for</Box>
```

Comment:
This should say `Question for` or something like that, no? Probably my bad when copy-pasting.